chalice/app.py

```diff
@@ -379,6 +429,20 @@ def _register_view(view_func):
             return view_func
         return _register_view
 
+    def _validate_route_content_types(self, name, content_types):
```

Comment:
Ah sure. I hadn't seen those, I just put it with the other route validation `ValueErrors` I may strip this check out entirely if it isn't deemed useful.